src/client/js/partials/breaches.js

```diff
@@ -43,6 +43,8 @@ function init () {
   statusFilter.addEventListener('change', handleEvent)
   breachesTable.addEventListener('change', handleEvent)
   document.body.addEventListener('email-added', handleEvent)
+
+  breachRows.forEach(elem => elem.addEventListener('toggle', handleEvent))
```

Comment:
@maxxcrawford would it be better to select an element higher in the DOM and add one listener there, vs. doing it this way?

Reply:
Seems fine to me, I don't expect major performance issues from just this - otherwise we can adjust.